### DISCUSSION TITLE
[Q] How to call predictor backend independently from trasformer

### DISCUSSION THREAD

**Discussion by cceyda** [kubeflow/kfserving#3330] [Q&A] [answered]
I'm following [this](https://kserve.github.io/website/0.11/modelserving/v1beta1/triton/torchscript/#add-transformer-to-the-inferenceservice) docs example on using triton as the predictor with a transformer.
Is it possible to call the predictor (in this case triton server) directly when deployed together with a transformer?
Use case: I wanna use the transformer for pre-post processing sometimes. But sometimes I want to call the triton server directly 

I tried a couple of things but I just can't figure out the right call address/port api signature. It always seems to route to the transformer.

**Comment by sivanantha321** [comment, marked as answer]
@cceyda Transformer and Predictor both have separate URLs. You can bypass the transformer by using the predictor URL. You can find it in the isvc status.
![Screenshot from 2023-12-02 11-33-58](https://github.com/kserve/kserve/assets/90966311/6b96cd17-5801-4c17-8db5-94d94cc3642b)
